tasks/main.yml

```diff
@@ -26,9 +26,10 @@
     name: "{{ item.name | default(undef(hint = 'You must provide a name')) }}"
     coe: "{{ item.coe | default(undef(hint = 'You must provide a coe')) }}"
     network_driver: "{{ item.network_driver | default(omit) }}"
-    master_lb_enabled: "{{ item.master_lb_enabled | default(omit) }}"
-    floating_ip_enabled: "{{ item.floating_ip_enabled | default(omit) }}"
+    master_lb_enabled: "{{ item.master_lb_enabled |default(omit) | bool }}"
```

Comment:
Does the bool filter do the right thing with an `omit` value? It's a bit weird, but it results in a sentinel string of the form `omit-gselkgkfnsld` which ansible uses to check for omitted arguments.